.github/workflows/desktop.yml

```diff
@@ -24,4 +24,4 @@ jobs:
       env:
         SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
         SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
-      run: mvn test -pl best-practice -Dtest=DesktopTests#loginWorks
+      run: mvn test -pl best-practice -Dtest=DesktopTests
```

Comment:
Having DesktopTests#loginWorks didn't seem to be working as expected and was throwing an error about no tests were executed so I thought I would put it back until we have a solution so that at least some desktop tests get executed